app/models/team_task.rb

```diff
@@ -98,6 +98,10 @@ def self.swap_order(task1, task2)
     task2_order = task2.order
     task1.update_column(:order, task2_order)
     task2.update_column(:order, task1_order)
+    # Apply new order to item annotations
+    fields = { order: true }
+    TeamTaskWorker.perform_in(1.second, 'update', task1.id, YAML::dump(User.current), YAML::dump(fields))
+    TeamTaskWorker.perform_in(1.second, 'update', task2.id, YAML::dump(User.current), YAML::dump(fields))
```

Comment:
We need to eventually replace how this Sidekiq worker receives parameters - it's a performance issue to pass in full serialized objects - @melsawy , can you please file a refactoring ticket for this?